.travis.yml

```diff
@@ -34,7 +34,14 @@ after_success:
       if grep -q "SNAPSHOT" version.sbt; then
         sbt ^^$SBT_VERSION publish;
       else
-        sbt ^^$SBT_VERSION release;
+        if [ "$SBT_VERSION" = "1.0.1" ]; then
+          sbt ^^$SBT_VERSION orgUpdateDocFiles;
+          git reset --hard HEAD;
+          git clean -f;
+          git checkout master;
+          git pull origin master;
+          sbt ^^$SBT_VERSION release;
+        fi
```

Comment:
Are we releasing only for1.0.1?

Reply:
It's a temporary guard band to only run `release` for 1 of the builds. `orgAfterCISuccess` needs to reworked a little bit to allow cross sbt publishing.

`"^ publishSigned"` in `releaseProcess` publishes for both sbt 1.0 and sbt 0.13
https://github.com/47deg/sbt-org-policies/blob/master/src/main/scala/sbtorgpolicies/settings/AllSettings.scala#L85